### DISCUSSION TITLE
[A-L] (2023/24) Foglio 2 - Esercizio 5

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#145] [Esercizi - Bertini] [answered]
![Es 5](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/2804e473-e9fa-436c-989f-215ef0c2e84f)

**Comment by daromacarlo** [comment]
![Immagine 2023-10-14 181727](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/8a75f94a-849a-41bf-97fe-348d39e8dcfb)
![Immagine 2023-10-14 181742](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/8af8618f-5a3c-4255-afae-10ca99d83bca)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/14c25d76-9152-4903-ae82-9c1a3ef2e52a)

**Comment by daromacarlo** [comment]
@CiottoloMaggico tu dici che sono 5 ragazzi compresi Alfredo e bianca?

**Comment by daromacarlo** [comment]
@CiottoloMaggico io ho capito che Alfredo e Bianca escono con altri 5 amici quindi in totale sono 7 ragazzotti.

**Comment by CiottoloMaggico** [comment]
@CarloDaRomadev effettivamente hai ragione, non so l'italiano

**Comment by Elia-Belli** [comment, marked as answer]
- $\Omega = \\{(w_1,..,w_7), w_1,..,w_5 = 1, ..,7, w_i\ distinti\\}, |\Omega|= 7!$
 $A = \\{Alfredo\ e\ Bianca\ sono\ adiacenti\ agli\ sgabelli\\}$
 Fisso Alfredo, se si trova al primo o all'ultimo sgabello Bianca ha solo una scelta per stargli vicino, altrimenti ne ha 2, in ogni caso distribuisco i restanti:
 $|A|=(2\cdot 1 + 5\cdot 2)\cdot 5!= 2 \cdot 6!$
 $\mathbb{P}(A)= \frac{2\cdot 6!}{7!}=2/7$

- $B = \\{Alfredo\ e\ Bianca\ sono\ adiacenti\ al\ tavolo\ tondo\\}$
Fisso Alfredo, adesso Bianca ha sempre 2 scelte per stargli vicino, poi distribuisco gli altri:
$|B|= 7\cdot 2\cdot 5! = 14 \cdot 5!$
$\mathbb{P}(B)= \frac{14\cdot 5!}{7!}= 1/3$
